pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "Toffy"
-version = "0.0.0"
+version = "0.2.1"
```

Comment:
The versioning schema is defined by the git tag of the release. Therefore we should not set the version number explicitly in the `pyproject.toml `file as it's generated by that tag during the build and upload to pypi process.

Reply:
In that case I think we can just close this PR and make a new release off of main?

Reply:
yup!